ibm/service/power/ibm_pi_constants.go

```diff
@@ -447,6 +481,12 @@ const (
 	vSCSI                     = "vSCSI"
 	Warning                   = "WARNING"
 
+	// Protocol Type
+	All  = "all"
+	ICMP = "icmp"
+	TCP  = "tcp"
+	UDP  = "udp"
```

Comment:
Those should go under the // Allowed values

Reply:
moved